keyboards/gon/nerd60/config.h

```diff
@@ -3,11 +3,11 @@
 #include "config_common.h"
 
 /* USB Device descriptor parameter */
-#define VENDOR_ID       0xFEED
-#define PRODUCT_ID      0x6FD7
-#define DEVICE_VER      0x0001
+#define VENDOR_ID       0x4E45
+#define PRODUCT_ID      0x3630
+#define DEVICE_VER      0x7283
```

Comment:
```suggestion
#define VENDOR_ID       0x4E45  // NE
#define PRODUCT_ID      0x3630  // 60
#define DEVICE_VER      0x0001
```

Reply:
Sorry FP, I'm still learning, especially with this whole github thing....